docs/docsite/rst/community/communication.rst

```diff
@@ -47,27 +47,27 @@ General channels
 Working groups
 --------------
 
-Many of our community Working Groups meet on Freenode IRC channels. If you want to get involved in a working group, join the channel where it meets or comment on the agenda.
+Many of our community `Working Groups <https://github.com/ansible/community/wiki#working-groups>`_ meet on Freenode IRC channels. If you want to get involved in a working group, join the channel where it meets or comment on the agenda.
 
-- `Amazon (AWS) Working Group <https://github.com/ansible/community/tree/master/group-aws>`_ - ``#ansible-aws``
-- `Ansible Lockdown Working Group <https://ansiblelockdown.io>`_ `gh/ansible/ansible-lockdown <https://github.com/ansible/ansible-lockdown>`_ - ``#ansible-lockdown``- Security playbooks/roles
+- `Amazon (AWS) Working Group <https://github.com/ansible/community/wiki/AWS>`_ - ``#ansible-aws``
+- `Ansible Lockdown Working Group <https://github.com/ansible/community/wiki/Lockdown>`_ | `gh/ansible/ansible-lockdown <https://github.com/ansible/ansible-lockdown>`_ - ``#ansible-lockdown``- Security playbooks/roles
 - `AWX Working Group <https://github.com/ansible/awx>`_ - ``#ansible-awx`` - Upstream Ansible Tower
-- `Azure Working Group <https://github.com/ansible/community/tree/master/group-azure>`_ - ``#ansible-azure``
-- `Community Working Group <https://github.com/ansible/community/tree/master/group-community>`_ - ``#ansible-community`` - Including Meetups
-- `Container Working Group <https://github.com/ansible/community/tree/master/group-container>`_ - ``#ansible-container``
-- `Contributor Experience Working Group <https://github.com/ansible/community/tree/master/group-contributor-experience>`_ - ``#ansible-community``
-- `Docker Working Group <https://github.com/ansible/community/issues/408>`_ - ``#ansible-devel``
```

Comment:
This link is still active.

Reply:
The new link for the Docker Working Group (https://github.com/ansible/community/wiki/Docker) has already a reference for https://github.com/ansible/community/issues/408 at "Docker meeting agenda".

Reply:
When I originally wrote this we didn't have a wiki page. Thanks for updating this.